src/entities/pair.test.ts

```diff
@@ -199,78 +199,120 @@ describe('Pair', () => {
       BLASTERSSellFeeBps
     )
 
-    describe('getOutputAmount', () => {
-      it('getOutputAmount for input token BLASTERS and output token BLAST', () => {
-        const reserveBlasterAmount = CurrencyAmount.fromRawAmount(BLASTERS, '10000')
-        const reserveBlastAmount = CurrencyAmount.fromRawAmount(BLAST, '10000')
-
-        const pair = new Pair(reserveBlasterAmount, reserveBlastAmount)
-
-        const inputBlastersAmount = CurrencyAmount.fromRawAmount(BLASTERS, '100')
-        const [outputBlastAmount] = pair.getOutputAmount(inputBlastersAmount)
-
-        // Theoretical amount out:
-        // (10000 * 997 * 100 * (1 - 3.5%) / (10000 * 1000 + 997 * 100 * (1 - 3.5%))) * (1 - 4%)
-        // = 91.48
-        //
-        // However in practice, we have round down of precisions in multiple steps
-        // hence the amount out will be slightly less than 91.48:
-        //
-        // inputAmount = 100
-        // percentAfterSellFeesInDecimal = fraction(9650, 10000)
-        // inputAmountAfterTax = 100 * fraction(9650, 10000) = 96.5 = 96 (rounded down)
-        // inputAmountWithFeeAndAfterTax = 96 * 997 = 95712
-        // numerator = 95712 * 10000 = 957120000
-        // denominator = 10000 * 1000 + 95712 = 10095712
-        // outputAmount = 957120000 / 10095712 = 94.8046061536 = 94 (rounded down)
-        // buyFeePercentInDecimal = fraction(400, 10000)
-        // percentAfterBuyFeesInDecimal = fraction(9600, 10000)
-        // outputAmountAfterTax = 94 * fraction(9600, 10000)
-        //                     = 94 * 0.96
-        //                     = 90.24
-        //                     = 90 (rounded down)
-        const expectedOutputBlastAmount = '0.00000000000000009'
-        expect(outputBlastAmount.toExact()).toEqual(expectedOutputBlastAmount)
+    let calculateFotFees: boolean = false
+
+    describe('when calculating FOT fees', () => {
+      beforeEach(() => {
+        calculateFotFees = true
+      })
+
+      describe('getOutputAmount', () => {
```

Comment:
I think we need two tests for `getOutputAmount` and two tests for `getInputAmount`, making it a total of 4.